pkg/pop/authnscheme.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// Disclaimer: This interface has been implemented for the usage of Azure Arc.
+
+package pop
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// type of a PoP token, as opposed to "JWT" for a regular bearer token
+const popTokenType = "pop"
+
+// PoPAuthenticationScheme is a PoP token implementation of the MSAL AuthenticationScheme interface.
```

Comment:
you should mention `PoPAuthenticationScheme is a PoP token implementation of the MSAL AuthenticationScheme interface used by the Azure Arc Platform team`

Reply:
Updated the text to be more specific